Kodilla-testing/src/main/java/com/kodilla/testing/forum/statistics/StatisticsExecutor.java

```diff
@@ -0,0 +1,44 @@
+package com.kodilla.testing.forum.statistics;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class StatisticsExecutor {
+    Statistics statistics;
```

Comment:
Pola jesli juz, to private! Ale tutaj nie potrzebujesz pola, gdyz poniewaz metoda przyjmuje argument typu Statistics.